### PR TITLE
make SingleParticle... examples 2D capable

### PR DESCRIPTION
**Pull request by PrometheusPi**
This pull request makes **all** SingleParticle... examples 2D capable.

### CODE REVIEW

File: examples/SingleParticleTest/include/particles/ParticlesInitOneParticle.hpp [hunk 1]
```diff
@@ -73,7 +73,9 @@ __global__ void kernelAddOneParticle(ParBox pb,
             setToDefault(forward(par));
         }
 
-        float3_X pos = float3_X(0.5, 0.5, 0.5);
+        floatD_X pos;
```
Comment (psychocoderHPC) [inline examples/SingleParticleTest/include/particles/ParticlesInitOneParticle.hpp]: please use  `floatD_X pos(floatD_X::create(0.5));`

File: examples/SingleParticleRadiationWithLaser/include/particles/ParticlesInitOneParticle.hpp [hunk 2]
```diff
@@ -77,8 +77,8 @@ __global__ void kernelAddOneParticle(ParBox pb,
         }
 
         floatD_X pos;
-        for (int i = 0; i < simDim; ++i)
```
Comment (psychocoderHPC) [inline examples/SingleParticleRadiationWithLaser/include/particles/ParticlesInitOneParticle.hpp]: please use  `floatD_X pos(floatD_X::create(0.5));`

File: examples/SingleParticleCurrent/include/particles/ParticlesInitOneParticle.hpp [hunk 3]
```diff
@@ -73,7 +73,10 @@ __global__ void kernelAddOneParticle(ParBox pb,
                 SetAttributeToDefault<bmpl::_1> > setToDefault;
             setToDefault(forward(par));
         }
-        float3_X pos = float3_X(LOCAL_POS_X, LOCAL_POS_Y, LOCAL_POS_Z);
+        float3_X posTmp = float3_X(LOCAL_POS_X, LOCAL_POS_Y, LOCAL_POS_Z);
```
Comment (psychocoderHPC) [inline examples/SingleParticleCurrent/include/particles/ParticlesInitOneParticle.hpp]: please write `floatD_X pos = float3_X(LOCAL_POS_X, LOCAL_POS_Y, LOCAL_POS_Z).shrink<simDim>();`
might be `floatD_X pos = float3_X(LOCAL_POS_X, LOCAL_POS_Y, LOCAL_POS_Z).template shrink<simDim>();`

File: examples/SingleParticleTest/include/particles/ParticlesInitOneParticle.hpp (hunk 1 above)
Comment (PrometheusPi) [inline examples/SingleParticleTest/include/particles/ParticlesInitOneParticle.hpp]: @psychocoderHPC I updated all files.

**Comment by psychocoderHPC** [comment]
+1

**Pull request merged**